companion/src/storage/mountlist.cpp

```diff
@@ -227,7 +227,7 @@
 
 #ifdef MOUNTED_GETMNTINFO
 
-# if ! HAVE_STRUCT_STATFS_F_FSTYPENAME
+# if ! HAVE_STRUCT_STATFS_F_FSTYPENAME && !defined(__clang__)
```

Comment:
~~Not your mistake, but loose space between `!` and `HAVE...`~~ Dont bother, this is thirdparty code.